api/src/main/java/org/openmrs/module/reporting/definition/library/implementerconfigured/BaseImplementerConfiguredDefinitionLibrary.java

```diff
@@ -125,6 +123,8 @@ private void loadDefinitions() {
 							log.warn("Invalid serialized definition at " + libraryNameSuffix + " in " + file
 									.getAbsolutePath(), ex);
 						}
+					} else if (filename.endsWith(".groovy")) {
+						definition = (Definition) new GroovyHelper().parseClassFromFileAndNewInstance(file);
```

Comment:
I wrote the code for autowiring (and I see a perhaps better way to let Spring entirely instantiate the bean) but I realize we're missing a key feature, which is that this only defines a _Definition_ not an _Evaluator_, which makes this entire commit pretty useless. I will move this conversation over to JIRA.